README.md

```diff
@@ -0,0 +1,121 @@
+# CMIS 1.1 compliant opencmis server
```

Comment:
we should remove this title from the toc (the toc need to be updated after this change ;-) )

```suggestion
# CMIS 1.1 compliant opencmis server <!-- omit in toc -->
```